index.html

```diff
@@ -3,9 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</CSD120-LiamLeung>
+    <title>CSD120 - Liam Leung</CSD120-LiamLeung>
 </head>
 <body>
-    "Hello, world!"
+    Hello, world!
```

Comment:
Don't include the quotation marks when I ask you to insert text in quotation marks